libqtile/backend/base.py

```diff
@@ -539,7 +543,6 @@ def clear(self, colour):
         self.set_source_rgb(colour)
         self.ctx.rectangle(0, 0, self.width, self.height)
         self.ctx.fill()
-        self.ctx.stroke()
```

Comment:
Was this doing nothing?

Reply:
I think so. `fill` fills the entire path and, as the `rectangle` is the entire drawer, there shouldn't be any pixels that haven't been painted by this.